### PR TITLE
Add tamil translations

### PR DESCRIPTION
**Pull request by ammahir**
Here is a checklist you should tick through before submitting a pull request: 
 - [ X] Implementation is clean
 - [X ] Code adheres to the existing coding standards; e.g. no curlies for one-line blocks, no redundant empty lines between methods or code blocks, spaces rather than tabs, etc.
 - [ ] No ReSharper warnings
 - [ X] There is proper unit test coverage
 - [ ] If the code is copied from StackOverflow (or a blog or OSS) full disclosure is included. That includes required license files and/or file headers explaining where the code came from with proper attribution
 - [X ] There are very few or no comments (because comments shouldn't be needed if you write clean code)
 - [ ] Xml documentation is added/updated for the addition/change
 - [ ] Your PR is (re)based on top of the latest commits from the `dev` branch (more info below)
 - [ ] Link to the issue(s) you're fixing from your PR description. Use `fixes #<the issue number>`
 - [ ] Readme is updated if you change an existing feature or add a new one
 - [ X] Run either `build.cmd` or `build.ps1` and ensure there are no test failures

### CODE REVIEW

**Comment by mohamedmahir** [comment]
Can someone help me on why build is failing?

File: src/Humanizer/Localisation/NumberToWords/TamilNumberToWordsConverter.cs [hunk 1]
```diff
@@ -0,0 +1,284 @@
+﻿using System;
+using System.Collections.Generic;
+//Author:Mohamed Mahir - mahir78@gmail.com
```
Comment (clairernovotny) [inline src/Humanizer/Localisation/NumberToWords/TamilNumberToWordsConverter.cs:3]: Pease remove this line

File: NuSpecs/Humanizer.Core.ta.nuspec [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.12">
+    <id>Humanizer.Core.ta</id>
+    <version>$version$</version>
+    <title>Humanizer Locale (ta)</title>
+    <authors>Mehdi Khalili, Claire Novotny, Mohamed Mahir</authors>
```
Comment (clairernovotny) [inline NuSpecs/Humanizer.Core.ta.nuspec:7]:
```suggestion
    <authors>Mehdi Khalili, Claire Novotny</authors>
```
Please change this

**Review by clairernovotny — CHANGES_REQUESTED** [review]
Thank you for this translation. We can merge this after the two small items are addressed. We let the commit history serve as credit.

File: src/Humanizer/Localisation/NumberToWords/TamilNumberToWordsConverter.cs (hunk 1 above)
Comment (clairernovotny) [inline src/Humanizer/Localisation/NumberToWords/TamilNumberToWordsConverter.cs:3]:
```suggestion
```

**Pull request merged**